### PR TITLE
DOCSTOOLS-703: move to uikit

### PR DESCRIPTION
**Pull request by yndx-birman**

### CODE REVIEW

**Review by martyanovandrey — APPROVED** [review]

File: src/demo/Components/DocPage/index.tsx [hunk 1]
```diff
@@ -61,10 +63,13 @@ const DocPageDemo = () => {
         console.log('Feedback:', data);
     }, []);
 
-    const onChangeBookmarkPage = useCallback((data: boolean) => {
-        setIsPinned(!data);
-        console.log(`This page pinned: ${data}`);
-    }, []);
+    const onChangeBookmarkPage = useCallback(
+        (data: boolean) => {
+            setIsPinned(data);
```
Comment (Lunatic174) [inline src/demo/Components/DocPage/index.tsx:68]: `setIsPinned(!data)` -> `setIsPinned(data)`
is it OK?

File: src/components/Controls/single-controls/SettingsControl/SettingsControl.tsx [hunk 2]
```diff
@@ -130,22 +133,28 @@ const SettingsControl = (props: ControlInnerProps) => {
...
-                                          active: textSize === textSizeKey,
                                       })}
+                                      view="flat"
                                       onClick={makeOnChangeTextSize(textSizeKey)}
                                   >
-                                      A
-                                  </ControlButton>
+                                      <Button.Icon
+                                          className={b('text-size-button-icon', {
+                                              active: textSize === textSizeKey,
```
[the remark this answers is not in the document]
Comment (yndx-birman) [reply src/components/Controls/single-controls/SettingsControl/SettingsControl.tsx:146]: there are no styles for `selected` modificator for `flat` view, only for `normal` view

File: src/demo/Components/DocPage/index.tsx (hunk 1 above)
Comment (yndx-birman) [reply src/demo/Components/DocPage/index.tsx:68]: yes, it was mistake in demo in storybook

File: src/demo/Components/DocPage/index.tsx [hunk 3]
```diff
@@ -61,10 +63,13 @@ const DocPageDemo = () => {
...
+            console.log(`This page pinned: ${data}`);
```
[the remark this answers is not in the document]
Comment (yndx-birman) [reply src/demo/Components/DocPage/index.tsx:69]: It's demo in storybook

File: styles/yfm.scss [hunk 4]
```diff
@@ -4,23 +4,23 @@
 .dc-doc-page .yfm {
```
[the remark this answers is not in the document]
Comment (yndx-birman) [reply styles/yfm.scss:4]: if only uikit dependency is not a problem for yfm

File: src/components/Controls/single-controls/LangControl.tsx [hunk 5]
```diff
@@ -55,6 +59,10 @@ const LangControl = (props: ControlInnerProps) => {
         return null;
     }
 
+    const itemHeight = 36;
```
[the remark this answers is not in the document]
Comment (yndx-birman) [reply src/components/Controls/single-controls/LangControl.tsx]: We have to set the size of the list item manually
I've made it a constant

File: src/demo/Components/DocPage/index.tsx [hunk 6]
```diff
@@ -61,10 +63,13 @@ const DocPageDemo = () => {
...
+        },
+        [isPinned],
```
[the remark this answers is not in the document]
Comment (yndx-birman) [reply src/demo/Components/DocPage/index.tsx]: not anymore

**Review by Lunatic174 — APPROVED** [review]

File: package.json [hunk 7]
```diff
@@ -92,9 +92,11 @@
     "@types/react": "16.14.23",
     "@types/react-dom": "16.9.14",
     "@yandex-cloud/eslint-config": "1.0.0",
+    "@yandex-cloud/i18n": "^0.6.0",
```
[the remark this answers is not in the document]
Comment (yndx-birman) [reply package.json:95]: It hasn't been planned, but lets add it to the issues

**Pull request merged**